src/cobalt.rs

```diff
@@ -281,6 +287,33 @@ fn create_rss(path: &str, dest: &Path, config: &Config, posts: &[Document]) -> R
         _ => Err(ErrorKind::ConfigFileMissingFields.into()),
     }
 }
+// creates a new jsonfeed file with the contents of the site blog
+fn create_jsonfeed(path: &str, dest: &Path, config: &Config, posts: &[Document]) -> Result<()> {
+    match (&config.name, &config.description, &config.link) {
+        (&Some(ref name), &Some(ref desc), &Some(ref link)) => {
+            trace!("Generating jsonfeed data");
+
+            let jsonitems = posts.iter().map(|doc| doc.to_jsonfeed(link)).collect();
+
+            let feed = Feed {
+              title: name.to_string(),
+              items: jsonitems,
+              home_page_url: Some(link.to_string()),
+              description: Some(desc.to_string()),
+              ..Default::default()
+            };
+
+            let jsonfeed_string = jsonfeed::to_string(&feed).unwrap();
+            let jsonfeed_path = dest.join(path);
+            let mut jsonfeed_file = try!(File::create(&jsonfeed_path));
+            try!(jsonfeed_file.write_all(&jsonfeed_string.into_bytes()));
+
+            info!("Created jsonfeed file at {}", jsonfeed_path.display());
```

Comment:
Nope:

info!("Created jsonfeed file at {}", jsonfeed_path);
    |                                                  ^^^^^^^^^^^^^ the trait `std::fmt::Display` is not implemented for `std::path::PathBuf`

This line was copied from the RSS feed creation :)